mirrord/kube/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "mirrord-kube"
+version.workspace = true
+authors.workspace = true
+description.workspace = true
+documentation.workspace = true
+readme.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+publish.workspace = true
+edition.workspace = true
+
+[features]
+default = []
+env_guard = []
+
+[dependencies]
+mirrord-config = { path = "../config"}
+mirrord-progress = { path = "../progress" }
+mirrord-protocol = { path = "../protocol" }
+
+actix-codec.workspace = true
+async-trait = "0.1"
+futures.workspace = true
+k8s-openapi.workspace = true
```

Comment:
I don't think we still need kube in the workspace?

Reply:
e2e tests also use kube and k8s-openapi so we can keep them but ill remove k8s-openapi from mirrord-layer

Reply:
put it in the specific cargo.tomls then